README.md

```diff
@@ -81,8 +81,8 @@ violations for your environment.
 --  Optional parameters:
 --    url - The base URL to iprepd (defaults to "http://localhost:8080/")
 --    timeout - The timeout for making requests to iprepd in milliseconds (defaults to 10)
---    cache_ttl - The iprepd response cache ttl in seconds (defaults to 30)
---    cache_buffer_count - Max number of entries allowed in the cache. (defaults to 200)
+--    cache_ttl - The iprepd response cache ttl in seconds (defaults to 60)
```

Comment:
Do we want to increase this one too? This means errors will also be cached for 60 seconds right?

Reply:
Hmm yeah, that is a good point. It may be worth having a separate ttl for caching errors?

Reply:
Yeah I think that sounds good 👌